### ISSUE TITLE
Cannot enable the secure dns from settings.

### ISSUE THREAD

**Issue by Rrauros** [flathub/com.microsoft.Edge#283]
It says "This setting is turned off for managed browsers." and its greyed out. 
This is not a problem when using the aur versions. 
This is kind of important feature since secure dns is the only way to bypass censorship for some of the website that governments are putting if you are not using vpn. 

![image](https://user-images.githubusercontent.com/62460316/226739292-a191ae8b-0333-4230-8321-8110eb4cbdc8.png)

**Comment by abuturabofficial** [comment]
Facing the same issue, as it is due to some policies set by Flatpak version of the Edge.
Another closed issue, said those policies can't be removed. So, what's the solution, really?

**Comment by abuturabofficial** [comment]
Maybe you can add DOH policy as well, which can allow this to happen? I don't know how to achieve this, but would love open PR, if someone can guide in the right direction?


> Facing the same issue, as it is due to some policies set by Flatpak version of the Edge. Another closed issue #255, said those policies can't be removed. So, what's the solution, really?

**Comment by abuturabofficial** [comment]
There are policy components for DNS-Over-HTTPS.

![edge_dns_policies](https://user-images.githubusercontent.com/68259771/236865066-122d0c61-10d9-47b7-a6b9-fb341d2982c7.png)

**Comment by Dyrimon** [comment]
I also couldn't find any solution. I've installed Brave and Chromium flatpak and they have the option to add custom DNS which works perfectly. If this is a flatpak issue surely there's some policy that can be configured in Flatseal?

**Comment by Dyrimon** [comment]
@Rrauros @abuturabofficial After digging through the system files, I've managed to find a workaround. Edit the `flatpak_policy.json` in `/var/lib/flatpak/app/com.microsoft.Edge/x86_64/stable/<random_string>/files/share/flatpak-edge/` and add `"DnsOverHttpsMode":automatic` inside the parentheses. Don't forget to add comma(,) at the end of the previous policy.

It should look like this:
```
{
  "DefaultBrowserSettingEnabled": false,
  "SmartScreenEnabled": false,
  "DnsOverHttpsMode":automatic
}
```
Now restart Edge and you should see it turned on. Now you can use any dns server you want. I tested with my NextDNS profile and it works.

You can add other policies if you want referencing this documentation: https://learn.microsoft.com/en-us/deployedge/microsoft-edge-policies

**Comment by evan-a-a** [comment]
Fixed in #388
